commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -402,12 +386,15 @@ def _prepare_segment_event_properties(in_order, total_amount, return_line_item_r
             else:
                 logger.info(f'[CT-{tag}] payment {psp_payment_id} refunded for message id: {message_id}')
 
-                total_amount = result.get('amount_in_cents')
+                total_in_dollars = result.get('amount_in_cents')
```

Comment:
We should also rename `amount_in_cents` to `amount_in_dollars` or simply `refund_amount` because we are not working with cents from start (the function returning amount to refund returns `cents_to_dollars(amount)`) which means the amount was never in cents during pipeline execution.

I am not sure why it was named in cents but if we dont rename now, we will be confused again in future and would have to dry run to know again.